py/server/setup.py

```diff
@@ -57,6 +57,7 @@ def normalize_version(version):
         'deephaven-plugin',
         'numpy',
         'pandas',
+        'pyarrow<10', # Turbodbc can't be built on MacOS 13.0.1 with pyarrow==10.0.1, filed an issue with the maintainer
```

Comment:
We should have a tracking ticket on our side to resolve this.

Reply:
I have filed a ticket with Turbodbc and have helped find a solution to the problem.  I will be notified once the fix is merged and tagged for release.